app/stacks/uk-west/back-office/storage-account.tf

```diff
@@ -28,14 +28,22 @@ resource "azurerm_storage_account" "back_office_documents" {
   }
 }
 
-resource "azurerm_storage_container" "back_office_documents_container" {
+resource "azurerm_storage_container" "back_office_document_service_uploads_container" {
   #TODO: Logging
   #checkov:skip=CKV2_AZURE_21 Logging not implemented yet
   name                  = "document-service-uploads"
   storage_account_name  = azurerm_storage_account.back_office_documents.name
   container_access_type = "private"
 }
 
+resource "azurerm_storage_container" "back_office_published_documents_container" {
+  #TODO: Logging
+  #checkov:skip=CKV2_AZURE_21 Logging not implemented yet
+  name                  = "published-documents"
+  storage_account_name  = azurerm_storage_account.back_office_documents.name
+  container_access_type = "private"
```

Comment:
This should be published right?

Reply:
So, I am thinking, for now I want to leave it "private". Because I would like to figure out at some point later how we can only allow public read access, but not public write access